aw_research/classify.py

```diff
@@ -423,7 +425,7 @@ def test_union_no_overlap():
 def get_events(since: datetime, end: datetime, include_smartertime='auto', include_toggl=None) -> List[Event]:
```

Comment:
Better to change `include_smartertime='auto'` to `False` than to pass it in the call on line 542.